negotiation/sessions/neural_session.py

```diff
@@ -129,6 +124,7 @@ def _create_batch(self):
                 'encoder_args': encoder_args,
                 'decoder_args': decoder_args,
                 }
+        pdb.set_trace()
```

Comment:
cleanup debug statements?

Reply:
done, that seems to be the only one left